<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDialog</class>
 <widget class="QDialog" name="OrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QLabel" name="aLabel">
         <property name="text">
          <string>A(单价: 10元)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bLabel">
         <property name="text">
          <string>B(单价: 20元)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cLabel">
         <property name="text">
          <string>C(单价: 30元)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dLabel">
         <property name="text">
          <string>D(单价: 40元)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="allLabel">
         <property name="text">
          <string>总计</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QLabel" name="aPriceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bPriceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cPriceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dPriceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="allPriceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
